### PR TITLE
refactor: home page refactor

### PR DESCRIPTION
**Pull request by storyworks**
# Description

Update home page to use our new project structure

# How should this PR be QA Tested?

- [x] HomePage still displays
- [x] New VR for home page

# Checklist

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] Any dependent changes have been merged into main

### CODE REVIEW

File: apps/watch/src/components/HomePage/HomePage.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  videos: VideoChildFields[]
+}
+
+export function HomePage({ videos }: HomePageProps): ReactElement {
+  return (
+    <PageWrapper hero={<HomeHero />}>
+      <ThemeProvider themeName={ThemeName.website} themeMode={ThemeMode.dark}>
+        <Container maxWidth="xxl" sx={{ paddingY: '4rem' }}>
+          <HomeVideos data={videos} />
+          <Box
```
Comment (storyworks) [inline apps/watch/src/components/HomePage/HomePage.tsx:24]: Added Intro text directly into home page for now.

File: apps/watch/pages/index.tsx [hunk 2]
```diff
@@ -1,23 +1,20 @@
...
-import { IntroText } from '../src/components/IntroText'
-import { VideoContentFields } from '../__generated__/VideoContentFields'
-import { VIDEO_CONTENT_FIELDS } from '../src/libs/videoContentFields'
+import { HomePage as VideoHomePage } from '../src/components/HomePage'
+import { VideoChildFields } from '../__generated__/VideoChildFields'
+import { VIDEO_CHILD_FIELDS } from '../src/libs/videoChildFields'
 
 export const GET_HOME_VIDEO = gql`
-  ${VIDEO_CONTENT_FIELDS}
+  ${VIDEO_CHILD_FIELDS}
```
Comment (csiyang) [inline apps/watch/pages/index.tsx:14]: Do you need to do a codegen because of this change?

File: apps/watch/src/components/HomePage/HomePage.tsx [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+interface HomePageProps {
+  videos: VideoChildFields[]
+}
+
+export function HomePage({ videos }: HomePageProps): ReactElement {
+  return (
+    <PageWrapper hero={<HomeHero />}>
+      <ThemeProvider themeName={ThemeName.website} themeMode={ThemeMode.dark}>
+        <Container maxWidth="xxl" sx={{ paddingY: '4rem' }}>
+          <HomeVideos data={videos} />
```
Comment (csiyang) [inline apps/watch/src/components/HomePage/HomePage.tsx:23]: Add a test that HomeVideo's gets rendered?

Comment (storyworks) [reply apps/watch/src/components/HomePage/HomePage.tsx:23]: Tested via VR?

Comment (storyworks) [reply apps/watch/src/components/HomePage/HomePage.tsx:23]: Tested via VR? I know the other pages have it.. it was left over from when we had logic checks that each element displayed based on what was passed to it.

**Pull request merged**